docs/site/Getting-started.md

```diff
@@ -43,10 +43,11 @@ Answer the prompts as follows:
 ? Application class name: StarterApplication
 ? Select features to enable in the project:
 ❯◉ Enable tslint: add a linter with pre-configured lint rules
- ◉ Enable prettier: add new npm scripts to facilitate consistent code formatting
- ◉ Enable mocha: install mocha to assist with running tests
+ ◉ Enable prettier: install prettier to format code conforming to rules
+ ◉ Enable mocha: install mocha to run tests
```

Comment:
nitpick: I personally like the previous version "install mocha to assist with running tests".
"install mocha to run tests" gives me an impression that some tests are created and ready to run, which is not quite true :-p

Reply:
I agree. @raymondfeng are you okay with changing it back?

Reply:
`install mocha to run tests` should be fine as `mocha` is test runner. I was motivated to trim the text so that more items can be displayed in one page.